packages/devextreme/testing/helpers/scheduler/helpers.js

```diff
@@ -270,10 +271,8 @@ class Appointment extends ClickElementWrapper {
     }
 
     click() {
-        const clock = sinon.useFakeTimers();
         this.getElement().trigger('dxclick');
-        clock.tick(300);
-        clock.restore();
```

Comment:
Why remove **restore** method?

Reply:
Looks like that now we use the one ```clock``` instance across the test case
So, it should restore at the end of the test I guess

@Alyar666, am I right? :)

Reply:
Yes, you are right